### ISSUE TITLE
MySQL: OperationalError: (1038, 'Out of sort memory, consider increasing server sort buffer size') 

### ISSUE THREAD

**Issue by grelleum** [nautobot/nautobot-plugin-ssot#92]
### Environment
* Python version: 3.8.13
* Nautobot version:  1.5.13
* nautobot-ssot version:  1.2.0

### Expected Behavior
When clicking `Plugins > Single Source of Truth > Dashboard` in the Nautobot UI, I expect to see the SSoT Dashboard.

### Observed Behavior
Page fails to load.  Debug mode presents a traceback with the exception:
```
OperationalError: (1038, 'Out of sort memory, consider increasing server sort buffer size')
```

### Steps to Reproduce
1. Deploy Nautobot using the MySQL backend database.
2. Run several large SSoT jobs.
3. Navigate to `Plugins > Single Source of Truth > Dashboard` in the Nautobot UI.

**Comment by grelleum** [comment]
Same error occurs when navigating to `Plugins > Single Source of Truth > Logs` in the Nautobot UI.

**Comment by Kircheneer** [comment]
@grelleum can this be closed now that #93 is merged?

**Comment by grelleum** [comment]
yes, thank you
